runtime/pom.xml

```diff
@@ -102,6 +102,59 @@
                     </annotationProcessorPaths>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-resources-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>process-codestart</id>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <phase>generate-resources</phase>
+                        <configuration>
+                            <escapeString>\</escapeString>
+                            <resources>
+                                <resource>
+                                    <directory>${project.basedir}/src/main/codestarts</directory>
+                                    <includes>
+                                        <include>**/codestart.yml</include>
+                                    </includes>
+                                    <filtering>true</filtering>
+                                </resource>
+                                <resource>
+                                    <directory>${project.basedir}/src/main/codestarts</directory>
+                                    <excludes>
+                                        <exclude>**/codestart.yml</exclude>
+                                    </excludes>
+                                    <filtering>false</filtering>
+                                </resource>
+                            </resources>
+                            <outputDirectory>${project.build.directory}/filtered-codestarts/codestarts</outputDirectory>
```

Comment:
Creates this one nested directory to much?
Shouldn't be filtered-codestarts be enough?

Reply:
No, the artifact should contain the `codestarts` folder, so the directory structure must be preserved